newt/image/encrypted.go

```diff
@@ -147,39 +147,9 @@ func unwrapPbes2Pbkdf2(param *pbkdf2Param, size int, iv []byte, hashNew hashFunc
 	plain := make([]byte, len(encrypted))
 	enc.CryptBlocks(plain, encrypted)
 
-	plain, err = checkPkcs7Padding(plain)
```

Comment:
It "works", but it then doesn't detect messages with corrupt padding. We really do need to verify that padding after performing the decryption.